fabric-object-builders-v1/src/main/java/net/fabricmc/fabric/api/entity/v1/FabricEntityTypeBuilder.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.entity.v1;
```

Comment:
The package doesn't correspond to the requirements outlined at https://github.com/FabricMC/fabric/blob/1.15/checkstyle.xml#L158 - please adjust